tests/unit/acquisition/function/test_greedy_batch.py

```diff
@@ -188,7 +188,7 @@ def test_fantasized_expected_improvement_builder_raises_for_invalid_fantasize_me
 
 def test_fantasized_expected_improvement_builder_raises_for_invalid_model() -> None:
     data = {
-        "OBJECTIVE": Dataset(tf.zeros([3, 2], dtype=tf.float64), tf.ones([3, 2], dtype=tf.float64))
+        "OBJECTIVE": Dataset(tf.zeros([3, 2], dtype=tf.float64), tf.ones([3, 1], dtype=tf.float64))
```

Comment:
wait - this should pass now?? the point of this test was to see if it catches data with a wrong observation shape.

Reply:
I thought the point of the test was to check for an invalid model (QuadraticMeanAndRBFKernel, which isn't a FastUpdateModel). The wrong observation shape results in an InvalidArgumentError exception rather than a NotImplementedError exception.

Reply:
(I've added another test for the observation shape)

Reply:
my bad - and thanks for improving this!!